### PR TITLE
LP-1780 Credentials and Discovery

### PR DESCRIPTION
**Pull request by aliassad**
Added a JWT key settings for LMS

### CODE REVIEW

File: course_discovery/settings/devstack.py [hunk 1]
```diff
@@ -22,6 +22,12 @@
 
 JWT_AUTH.update({
     # Must match public signing key used in LMS.
+    'JWT_ISSUERS':
+        [{
+            "AUDIENCE": "lms-key",
+            "ISSUER": "http://local.philanthropyu.org:18000/oauth2", 
+            "SECRET_KEY": "lms-secret"
+        }],
```
Comment (muhammadzeshanarif) [inline course_discovery/settings/devstack.py]: move this above this comment `# Must match public signing key used in LMS.` and properly format

Comment (aliassad) [reply course_discovery/settings/devstack.py]: Done

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

**Review by muhammadzeshanarif — APPROVED** [review]
LGTM 👍

**Pull request merged**